setup.py

```diff
@@ -17,7 +17,8 @@
     python_requires  = '>=3.5',
     install_requires = ['pynvim', 'psutil', 'setuptools'],
     entry_points     = {
-        'console_scripts': ['nvr = nvr.nvr:main']
+        'console_scripts': ['nvr = nvr.nvr:main'],
+        'gui_scripts': ['gnvr = nvr.nvr:main']
```

Comment:
What is this for? Why is it needed?

Reply:
`gui_scritpts` will create an binary `gnvr.exe`, the difference from `nvr.exe` is it wouldn't bring up the terminal. I guess on Windows there is a clear line between "console program" and "gui program".  While on *nix, `gnvr` and `nvr` are just the same script. The user case is mentioned as above.

Reply:
What happens if you use `gnvr.exe` with `NVR_CMD=nvim` (the default)?

I think `gui_scripts` is only needed for Windows, but the question is: Do we even need `console_scripts`?

Reply:
Set `NVR_CMD=nvim`, and type `gnvr.exe file` in your terminal, it will still work(partly), except that it will NEVER echo any messages, ever it encounts huge errors.

We'll still need `console_scripts`, they are respectively for different user cases.